modules/installation-guide/examples/system-variables.adoc

```diff
@@ -226,15 +226,16 @@
- `+CHE_WORKSPACE_DEVFILE_DEFAULT__EDITOR_PLUGINS+`,"`+eclipse/che-machine-exec-plugin/7.23.2+`","Default Plugins which should be provisioned for Default Editor. All the plugins from this list that are not explicitly mentioned in the user-defined devfile will be provisioned but only when the default editor is used or if the user-defined editor is the same as the default one (even if in different version). Format is comma-separated `pluginPublisher/pluginName/pluginVersion` values, and URLs. For example: eclipse/che-theia-exec-plugin/0.0.1,eclipse/che-theia-terminal-plugin/0.0.1,https://cdn.pluginregistry.com/vi-mode/meta.yaml If the plugin is a URL, the plugin's meta.yaml is retrieved from that URL." 
+ `+CHE_WORKSPACE_DEVFILE_DEFAULT__EDITOR+`,"`+eclipse/che-theia/latest+`","Default Editor that should be provisioned into Devfile if there is no specified Editor Format is `editorPublisher/editorName/editorVersion` value. `NULL` or absence of value means that default editor should not be provisioned." 
+ `+CHE_WORKSPACE_DEVFILE_DEFAULT__EDITOR_PLUGINS+`,"`+eclipse/che-machine-exec-plugin/latest+`","Default Plugins which should be provisioned for Default Editor. All the plugins from this list that are not explicitly mentioned in the user-defined devfile will be provisioned but only when the default editor is used or if the user-defined editor is the same as the default one (even if in different version). Format is comma-separated `pluginPublisher/pluginName/pluginVersion` values, and URLs. For example: eclipse/che-theia-exec-plugin/0.0.1,eclipse/che-theia-terminal-plugin/0.0.1,https://cdn.pluginregistry.com/vi-mode/meta.yaml If the plugin is a URL, the plugin's meta.yaml is retrieved from that URL." 
  `+CHE_WORKSPACE_PROVISION_SECRET_LABELS+`,"`+app.kubernetes.io/part-of=che.eclipse.org,app.kubernetes.io/component=workspace-secret+`","Defines comma-separated list of labels for selecting secrets from a user namespace, which will be mount into workspace containers as a files or env variables. Only secrets that match ALL given labels will be selected." 
- `+CHE_WORKSPACE_DEVFILE_ASYNC_STORAGE_PLUGIN+`,"`+eclipse/che-async-pv-plugin/nightly+`","Plugin is added in case async storage feature will be enabled in workspace config and supported by environment" 
- `+CHE_INFRA_KUBERNETES_ASYNC_STORAGE_IMAGE+`,"`+quay.io/eclipse/che-workspace-data-sync-storage:latest+`","Docker image for the {prod-short} async storage" 
+ `+CHE_WORKSPACE_DEVFILE_ASYNC_STORAGE_PLUGIN+`,"`+eclipse/che-async-pv-plugin/latest+`","Plugin is added in case async storage feature will be enabled in workspace config and supported by environment" 
+ `+CHE_INFRA_KUBERNETES_ASYNC_STORAGE_IMAGE+`,"`+quay.io/eclipse/che-workspace-data-sync-storage:0.0.1+`","Docker image for the {prod-short} async storage" 
```

Comment:
latest is newer than 0.0.1. Why use old one and have to update it manually? 

https://quay.io/repository/eclipse/che-workspace-data-sync-storage?tag=latest&tab=tags

Reply:
Then we need to update it in the source in: https://github.com/eclipse/che/blob/7.25.x/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/multiuser.properties

Reply:
@nickboldt target is 7.25.x, are you sure the version should be updated in  https://github.com/eclipse/che/blob/7.25.x/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/multiuser.properties ?

Reply:
no idea, I was just pointing out a discrepancy. @skabashnyuk might be able to help you here